app/templates/welcome.js

```diff
@@ -64,6 +64,14 @@ module.exports = function(state, emit) {
     if (file.size === 0) {
       return;
     }
+    if (file.size > 2000000000) {
+      window.alert(
```

Comment:
please use `state.translate('fileTooBig', { size: bytes(MAXFILESIZE) })` instead of the hardcoded string. Where `bytes` is from `app/utils.js`